crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -154,6 +169,16 @@ impl BundleFactory {
             }
             Err(SizedBundleError::NotEnoughSpace(seq_action)) => {
                 // if the bundle is full, flush it and start a new one
+                metrics::counter!(crate::metrics_init::BUNDLES_TOTAL_COUNT).increment(1);
+
+                #[allow(clippy::cast_precision_loss)]
```

Comment:
this looks a bit odd. I am not sure if there is a much more cleaner way to do this?

Reply:
you can to `self.curr_bundle.get_size().try_into()` or perhaps `TryInto::<f64>::try_into(self.curr_bundle.get_size())`  since you might need to specify the type   - do you expect the current bundle size to always fit into a f64 or no? same for the other values being casted

Reply:
it should fit into a 64 byte number for sure!

Reply:
`clippy::cast_precision_loss` is fine. Just add a comment above why you allow it.

We never want to fail on metrics. It's not material